catalog/tests/dags/providers/provider_api_scripts/test_wikimedia_commons.py

```diff
@@ -416,7 +416,7 @@ def test_create_meta_data_scrapes_text_from_html_description(wmc):
     media_data = _get_resource_json("image_data_html_description.json")
     expect_description = (
         "Identificatie Titel(s):  Allegorie op kunstenaar Francesco Mazzoli, "
-        "bekend als Parmigianino"
+        "bekend also Parmigianino"
```

Comment:
Considering that this is the name of an artwork, a `# noqa` (or equivalent) would be more appropriate here.

Reply:
Oops! This was one that got automatically caught. codespell doesn't have `noqa`, but we can add it to the `ignore_lines.txt` :+1: